src/lib.rs

```diff
@@ -292,6 +292,14 @@ macro_rules! declare_types {
     { } => { };
 }
 
+#[doc(hidden)]
+#[macro_export]
+macro_rules! neon_stringify {
```

Comment:
Was this necessary? It doesn't seem like it should use `stringify!` since `use std` is implied.

Reply:
class_definition use `stringify!`. According to https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html#macros-using-local_inner_macros we need add a local macro that wraps `stringify!` since `stringify!` isn't in the neon crate. Without this code that uses class_definition won't compile.

Reply:
`test/dynamic/native/src/js/classes.rs` uses declare_types! to declare classes. It has the error message "could not find 'stringify' in '$crate' when I compile it without wrapping `stringify!` in a local macro.